src/plugin.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import {definePlugin, isObjectInputProps, SchemaType} from 'sanity'
+import {AprimoCDNWidget, AprimoConfig} from './components/AprimoCDNWidget'
+import {AprimoCDNAsset, AprimoCDNAssetSchema} from './schema/AprimoCDNAsset'
+
+export const aprimoPlugin = definePlugin((config: Partial<AprimoConfig>) => {
+  const reqConfig: AprimoConfig = {
+    tenantName: '',
+    ...config,
+  }
+
+  return {
+    name: 'aprimo-assets',
+    schema: {
+      types: [AprimoCDNAssetSchema],
```

Comment:
Since we are removing a schema type, upgrading to this version may leave a lot of unknown fields of type `aprimo.asset` set in the users Studio. We should at least warn about this in the README and release notes. We also need to make this a major version update.